src/server/api/referenceGenomes.js

```diff
@@ -27,5 +27,41 @@ export default function refGenomeRouter() {
         })
     );
 
+    logger.info("adding the Admin GET /admin/reference-genomes");
+    api.get(
+        "/admin/reference-genomes",
```

Comment:
Small thing, but since this isn't actually an admin endpoint, I would remove `/admin` from the logger and the `api.get(...)`

Reply:
Actually, `GET` `reference-genomes` is not under `admin` but I added separate `handler` under `admin` cos admins can see deleted `reference genomes`. May be I will just query-params alone to do that distinction.